plugins/CoreAdminHome/CustomLogo.php

```diff
@@ -163,6 +163,16 @@ protected static function rewritePath($path)
         return SettingsPiwik::rewriteMiscUserPathWithInstanceId($path);
     }
 
+    public static function hasUserLogo()
+    {
+        return file_exists(Filesystem::getPathToPiwikRoot() . '/' . CustomLogo::getPathUserLogo());
```

Comment:
BTW: Here you could use `PIWIK_INCLUDE_PATH . CustomLogo::getPathUserLogo()` and same below